docker/Tomcat/webapp/Dockerfile

```diff
@@ -20,8 +20,10 @@ RUN set -eux && \
 USER tomcat
 
 # Copy environment configuration
-COPY --chown=tomcat src/scripts/catalinaAdditional.properties /tmp
-RUN cat /tmp/catalinaAdditional.properties >> /usr/local/tomcat/conf/catalina.properties  && rm -f /tmp/catalinaAdditional.properties
+COPY --chown=tomcat src/scripts/ /scripts/
+RUN cat /scripts/catalinaAdditional.properties >> /usr/local/tomcat/conf/catalina.properties  && \
+    rm -f /tmp/catalinaAdditional.properties && \
```

Comment:
`/tmp/catalinaAdditional.properties` has been changed to be in `/scripts/` but you still try to delete it at the old location